File: q/c/submissions/nazar/problem4.c
#include<stdio.h>
int main()
{
int n, sum=0;int num;
while(n!=0)
{
num=n;
n=n%10;
sum=sum+n;
n=n/10;
}
sum=n;
if(sum==1)
printf("no");
else
printf("yes");
return 0;
}

File: round 1/Subhro@yon1/problem4.java
import java.io.*;
public class problem4
{
public static void main(String[]args)throws IOException
{
BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
int num;
int sum=0,sum1=0,d,d1;
num = Integer.parseInt(br.readLine());
d=num%10;
sum=sum+d;
num=num/10;
d1=sum%10;
sum1=sum1+d1;
sum=sum/10;
if(sum1==1)
System.out.println("yes");
else
System.out.println("no");
}
}

